prog/dftb+/lib_math/eigensolver.F90

```diff
@@ -2228,15 +2228,15 @@ end subroutine dblecmplx_zhbgv
     if (present(vl)) then
       jobvl = 'V'
       ldvl = size(vl, dim=1)
-      @:ASSERT(all(shape(vr)>=[n,n]))
+      @:ASSERT(all(shape(vl)>=[n,n]))
```

Comment:
@bhourahine Could you please confirm, that the change is OK? (Otherwise the ASSERT is triggered in the pp-RPA examples.)

Reply:
Can confirm that this is now correct.